src/js/view/controls/settings-menu.js

```diff
@@ -64,6 +64,18 @@ export function setupSubmenuListeners(settingsMenu, controlbar, viewModel, api)
         }
     };
 
+    const changeAutoLabel = function (quality, qualitySubMenu, currentQuality) {
```

Comment:
If you haven't changed this method, can you put it back where it was? It does not need to be defined before `onAudiotracksChanged`. It is available when `onAudiotracksChanged` is run.